.github/labeler.yml

```
@@ -0,0 +1,30 @@
+- name: Labeler
+  uses: actions/labeler@v3
+lang/ko:
+- content/ko/*
+lang/bn:
+- content/bn/*
+lang/en:
+- content/en/*
+lang/es:
+- content/es/*
+lang/ar:
+- content/ar/*
+lang/de:
+- content/de/*
+lang/pt:
+- content/pt-br/*
+lang/it:
+- content/it/*
+lang/hi:
+- content/hi/*
+lang/de:
+- content/de/*
+lang/ar:
+- content/ar/*
+lang/fr:
+- content/fr/*
+
+
```

```suggestion
lang/zh:
- content/zh/*
```
Could you add this configuration too ? (`lang/zh` has been added recently)